packages/platforms/browser/lib/auto-instrumentation/resource-load-plugin.ts

```diff
@@ -53,8 +53,10 @@ export class ResourceLoadPlugin implements Plugin<BrowserConfiguration> {
 
         const parentContext = this.spanContextStorage.first
 
-        if (parentContext) {
-          const url = new URL(entry.name)
+        const networkRequestInfo = configuration.networkRequestCallback({ url: entry.name, type: entry.initiatorType })
+
+        if (parentContext && networkRequestInfo) {
+          const url = new URL(networkRequestInfo.url)
```

Comment:
What if `networkRequestInfo.url` isn't a string?

Reply:
I suppose we probably need to log a warning and not create the span... we could add a try/catch to the URL constructor which would also gracefully handle an incomplete URL?

Reply:
actually, that's a little over-complex and if we're monkey patching a native request, the user will likely already have thrown an error in their application from trying to make a request with an invalid endpoint. updated!

Reply:
The logic should be the same as in #263

A try/catch for invalid URLs makes sense too, though the URL constructor expects a full URL so it can't just be a relative path. Should the name still use `entry.name`?

Reply:
I don't we should directly use `entry.name` as that's the property that the user may have removed personal information from

Reply:
i.e. path parameters with a user id, or an api key.

Reply:
OK, makes sense; we'll need to handle `networkRequestInfo.url` being invalid here then